app/models/manageiq/providers/ibm_cloud/power_virtual_servers/cloud_manager/provision/cloning.rb

```diff
@@ -16,16 +16,17 @@ def prepare_for_clone_task
       specs['profile_id']   = get_option_last(:sys_type)
       specs['ssh_key_name'] = chosen_key_pair unless chosen_key_pair == 'None'
     else
-      specs['server_name']   = get_option(:vm_target_name)
-      specs['memory']        = get_option_last(:vm_memory).to_i
-      specs['migratable']    = get_option_last(:migratable) == 1
-      specs['processors']    = get_option_last(:entitled_processors).to_f
-      specs['proc_type']     = get_option_last(:instance_type)
-      specs['pin_policy']    = get_option_last(:pin_policy)
-      specs['replicants']    = 1 # TODO: we have to use this field instead of what 'MIQ' does
-      specs['key_pair_name'] = chosen_key_pair unless chosen_key_pair == 'None'
-      specs['storage_type']  = get_option_last(:storage_type)
-      specs['sys_type']      = get_option_last(:sys_type)
+      specs['server_name']     = get_option(:vm_target_name)
+      specs['memory']          = get_option_last(:vm_memory).to_i
+      specs['migratable']      = get_option_last(:migratable) == 1
+      specs['processors']      = get_option_last(:entitled_processors).to_f
+      specs['proc_type']       = get_option_last(:instance_type)
+      specs['pin_policy']      = get_option_last(:pin_policy)
+      specs['placement_group'] = get_option_last(:placement_group)
```

Comment:
We'll need to update the provisioning form to include `placement_group` before we can add it to the tasks here. Let's save the provisioning changes for a later PR.

Reply:
sure, I can remove ut from here